static/js/admin/src/ResultObjectActionMenu.js

```diff
@@ -190,13 +190,18 @@ define('ResultObjectActionMenu', [ 'jquery', 'jquery-ui', 'StringUtilities',  'A
 		}
 
 		// Set/Update permission actions
-		items["seppermission"] = "";
+		if ($.inArray('assignStaffRoles', metadata.permissions) !== -1 || (metadata.type !== 'AdminUnit' && $.inArray('changePatronAccess', metadata.permissions) !== -1)) {
```

Comment:
this looks fine for the most part, but mind if I suggest determining permissions once in variables and then reusing those rather than duplicating the checks?
```
let canChangePatronAccess = // whatever
let canAssignStaffRoles = // whatever

if (canChangePatronAccess || canAssignStaffRoles) {
```